pkg/config/polling_manager_test.go

```diff
@@ -44,6 +44,18 @@ func newExecGroup() *utils.ExecGroup {
 	return utils.NewExecGroup(context.Background())
 }
 
+// assertion method to wait for config or err for a specified period of time.
+func waitForConfigOrCancelTimeout(t *testing.T, configManager ProjectConfigManager, checkError bool) {
```

Comment:
I would like to hear if we can introduce `onReady` like other SDKs. Then this method is not needed.

Reply:
not needed. got mike's feedback.